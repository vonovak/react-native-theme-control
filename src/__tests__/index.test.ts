import * as RNTC from '../index';

it('sanity check for exported methods', () => {
  expect(RNTC).toMatchSnapshot();
});
