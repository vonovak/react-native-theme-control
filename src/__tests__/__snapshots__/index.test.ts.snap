// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sanity check for exported methods 1`] = `
{
  "AppBackground": [Function],
  "NavigationBar": [Function],
  "SystemBars": [Function],
  "ThemeAwareStatusBar": [Function],
  "getThemePreference": [Function],
  "setAppBackground": [Function],
  "setNavbarAppearance": [Function],
  "setThemePreference": [Function],
  "useThemePreference": [Function],
}
`;
